src/Network.cpp

```diff
@@ -869,17 +869,28 @@ T relative_difference(T a, T b) {
 
 void compare_net_outputs(std::vector<float>& data,
                          std::vector<float>& ref) {
+    // Divide by an additional 2 as we're comparing 2 heads at a time
+    constexpr int64 min_correct_expansions = SELFCHECK_MIN_EXPANSIONS / SELFCHECK_PROBABILITY / 2;
+    static_assert(min_correct_expansions > 0, "Increase minimal nof expansions");
+    static int64 num_expansions{min_correct_expansions};
```

Comment:
This could be problematic for multi-threaded evaluation.

Reply:
Right! I guess this should be atomic.

Reply:
Right! I guess this should be atomic.